src/codec/field_codec.h

```diff
@@ -27,227 +27,16 @@
 #include "base/glog_wapper.h"
 #include "base/strings.h"
 #include "boost/lexical_cast.hpp"
-#include "codec/memcomparable_format.h"
 #include "proto/type.pb.h"
 #include "sdk/base.h"
 
-namespace openmldb {
-namespace codec {
```

Comment:
do not delete the namespace declaration

Reply:
error fixed, sorry for that mistaken deletion